daemon/container.go

```diff
@@ -237,6 +237,21 @@ func (daemon *Daemon) verifyContainerSettings(hostConfig *containertypes.HostCon
 				return nil, err
 			}
 		}
+
+		// Validate the healthcheck params of Config
+		if config.Healthcheck != nil {
+			if config.Healthcheck.Interval < 0 {
```

Comment:
Sure, consistent is good, but the daemon side is the "important" piece.
I just think if we are going to add a validation, let's make sure it's not letting non-sensical values.